lib/go-atscfg/remapdotconfig.go

```diff
@@ -984,24 +984,24 @@ func remapFilterDSes(server *Server, dss []DeliveryServiceServer, dses []Deliver
 		if ds.LastHeaderRewrite == nil {
 			ds.LastHeaderRewrite = util.StrPtr("")
 		}
-		if ds.XMLID == nil {
+		if &ds.XMLID == nil {
 			warnings = append(warnings, "got Delivery Service with nil XMLID, skipping!")
 			continue
 		} else if ds.Type == nil {
-			warnings = append(warnings, "got Delivery Service '"+*ds.XMLID+"'  with nil Type, skipping!")
+			warnings = append(warnings, "got Delivery Service '"+ds.XMLID+"'  with nil Type, skipping!")
 			continue
-		} else if ds.DSCP == nil {
-			warnings = append(warnings, "got Delivery Service '"+*ds.XMLID+"'  with nil DSCP, skipping!")
+		} else if &ds.DSCP == nil {
+			warnings = append(warnings, "got Delivery Service '"+ds.XMLID+"'  with nil DSCP, skipping!")
```

Comment:
Instead of checking `&ds.DSCP == nil`, check `ds.DSCP == ""`.

Reply:
fixed